handlers/metrics.go

```diff
@@ -0,0 +1,52 @@
+package handlers
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	RedirectHandlerRedirectCountMetric = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "router_redirect_handler_redirect_count",
```

Comment:
```
```suggestion
			Name: "router_redirect_handler_redirect_total",
```

Reply:
Resolved in 6b1e0e8